aggsender/flows/flow_base.go

```diff
@@ -356,39 +357,17 @@ func (f *baseFlow) getImportedBridgeExits(
 	// Build unclaim counts by GlobalIndex
 	unclaimCnt := make(map[string]int)
```

Comment:
WDYT, maybe it would be easier not to index by strings but by raw global index `big.Int`? Although I'm unsure what is more efficient, it might be by strings...

```suggestion
	unclaimCnt := make(map[*big.Int]int)
```

Reply:
done